zipline/finance/performance/period.py

```diff
@@ -193,15 +227,14 @@ def position_tracker(self, obj):
         # we only calculate perf once we inject PositionTracker
         self.calculate_performance()
 
-    def rollover(self):
-        self.starting_value = self.ending_value
-        self.starting_exposure = self.ending_exposure
-        self.starting_cash = self.ending_cash
-        self.period_cash_flow = 0.0
-        self.pnl = 0.0
-        self.processed_transactions = {}
-        self.orders_by_modified = {}
-        self.orders_by_id = OrderedDict()
+    def rollover(self, capital_change=0):
```

Comment:
Can we make `capital_change`'s default `None` instead of 0, to make it clear that there are cases where the `ending_cash` is change or not?
And then make the change to cash as:
```
if capital_change is not None:
    self.ending_cash += capital_change
```

Though having said that, if we only change the value in the case of `if is_interday` in `process_capital_changes`, I think it would be more clear to make `process_capital_changes` do the processing of the capital_change in the scope of that function, i.e..:

```
if is_interday:
   # Change comes between days
   self.ending_cash += capital_change
   self.todays_performance.rollover() # or self.todays_performance.start_new_subperiod()
```